gasmodel/Pact/Core/GasModel/BuiltinsGas.hs

```diff
@@ -77,10 +97,24 @@ benchArithBinOp' growthFactor op pdb =
     [ runNativeBenchmark pdb title [text|($op $x $x.0)|] | (title, x) <- vals ]
   ]
   where
-  vals = take 3 $ enumExpText 1000 growthFactor
+  vals = take 3 $ enumExpText 1000 1000000
```

Comment:
Maybe we should use `NumericUnderscores` and `1_000_000`?

```suggestion
  vals = take 3 $ enumExpText 1000 1_000_000
```

Reply:
Good idea! I'll make a pass over the code to see what other greater-than-1000 numbers are there.

Reply:
Went through all of it and changed all numbers bigger than 1000 to have `_` separators. Great suggestion, thanks!